renderdoc/driver/vulkan/vk_layer.cpp

```diff
@@ -163,7 +163,7 @@ VkResult VKAPI_CALL hooked_vkCreateInstance(const VkInstanceCreateInfo *pCreateI
                                             const VkAllocationCallbacks *pAllocator,
                                             VkInstance *pInstance)
 {
-  WrappedVulkan *core = new WrappedVulkan("");
+  WrappedVulkan *core = new WrappedVulkan(0);
```

Comment:
So the crash happens when I try to save a capture to my pc.
Error   - Can't open capture file 'C:\Users\marton.tamas\Desktop\bla.rdc' for read - errno 2

Thread 4 "renderdoccmd" received signal SIGTRAP, Trace/breakpoint trap.
[Switching to Thread 0x7ffff3ab0700 (LWP 29744)]
0x00007ffff65c5428 in __GI_raise (sig=5) at ../sysdeps/unix/sysv/linux/raise.c:54
54      ../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
(gdb) bt
#0  0x00007ffff65c5428 in __GI_raise (sig=5) at ../sysdeps/unix/sysv/linux/raise.c:54
#1  0x00007ffff778d788 in Serialiser::Serialiser (this=0x7ffff3aaf9c0, path=0x7fffec071a30 "C:\\Users\\marton.tamas\\Desktop\\bla.rdc", mode=Serialiser::READING, debugMode=true, sizeHint=131072)
    at /home/plato/marton.tamas/renderdoc/renderdoc/serialise/serialiser.cpp:662
#2  0x00007ffff7710ebf in RenderDoc::FillInitParams (this=0x7ffff7dcb860 <RenderDoc::Inst()::realInst>, logFile=0x7fffec071a30 "C:\\Users\\marton.tamas\\Desktop\\bla.rdc", driverType=@0x7ffff3aafb94: RDC_Unknown, driverName="",
    fileMachineIdent=@0x7ffff3aafbf0: 0, params=0x0) at /home/plato/marton.tamas/renderdoc/renderdoc/core/core.cpp:755
#3  0x00007ffff772bb66 in ActiveRemoteClientThread (data=0x6535f0) at /home/plato/marton.tamas/renderdoc/renderdoc/core/remote_server.cpp:375
#4  0x00007ffff78410b9 in Threading::sThreadInit (init=0x653610) at /home/plato/marton.tamas/renderdoc/renderdoc/os/posix/posix_threading.cpp:118
#5  0x00007ffff61766ba in start_thread (arg=0x7ffff3ab0700) at pthread_create.c:333
#6  0x00007ffff66973dd in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:109